src/backend/base/langflow/components/nvidia/nemo_evaluator.py

```diff
@@ -28,44 +14,39 @@ class NVIDIANeMoEvaluatorComponent(Component):
     beta = True
 
     # Endpoint configuration
-    # This assumes that the inference URL is a Kubernetes service in the same namespace as the evaluator
+    endpoint = os.getenv("NVIDIA_EVALUATOR_BASE_URL", "http://35.223.81.140:11000")
+    model_endpoint = os.getenv("NVIDIA_MODELS_BASE_URL", "http://35.223.81.140:10000")
+    datastore_base_url = os.getenv("NVIDIA_DATA_STORE_BASE_URL", "http://35.223.81.140:8000")
+    url = f"{endpoint}/v1/evaluations"
+    model_url = f"{model_endpoint}/v1/models"
     inference_url = "http://nemo-nim.default.svc.cluster.local:8000/v1"
 
-    headers = {"accept": "application/json", "Content-Type": "application/json"}
+    headers = {
+        'accept': 'application/json',
+        'Content-Type': 'application/json'
+    }
 
     # Define initial static inputs
     inputs = [
-        StrInput(
-            name="evaluator_base_url",
-            display_name="NVIDIA NeMo Evaluator Base URL",
-            info="The base URL of the NVIDIA NeMo Evaluator API.",
-        ),
-        StrInput(
-            name="model_base_url",
-            display_name="NVIDIA NeMo Model URL",
-            info="The base URL of the NVIDIA NIM API to obtain models that can be evaluated.",
-        ),
         DropdownInput(
             name="000_llm_name",
             display_name="LLM Name",
             info="Select the model for evaluation",
             options=[],  # Dynamically populated
-            refresh_button=True,
+            refresh_button=True
         ),
         StrInput(
             name="001_tag",
             display_name="Tag",
-            info="Any user-provided value. Generated results will be stored in the NeMo Data Store under this name.",
-            value="default",
-            required=True,
```

Comment:
I made this required, because if you put empty string for this value, the API returns HTTP 422 UNPROCESSABLE CONTENT.